internal/datacoord/segment_manager.go

```diff
@@ -184,7 +185,7 @@ func defaultAllocatePolicy() AllocatePolicy {
 
 func defaultSegmentSealPolicy() []segmentSealPolicy {
 	return []segmentSealPolicy{
-		sealByLifetimePolicy(Params.DataCoordCfg.SegmentMaxLifetime.GetAsDuration(time.Second)),
+		sealByLifetimePolicy(Params.DataCoordCfg.SegmentMaxLifetime.GetAsDuration(time.Hour)),
```

Comment:
Previously configured value is 86400.
If the default unit is changed to Hour, is the lifespan of the segment too long?

Reply:
the default time is https://github.com/milvus-io/milvus/blob/a4e6c2ce79cfe47c5fe161388f6e1acf2fd3c1da/internal/util/paramtable/component_param.go#L1319 , the unit should be unified,  we should not change time unit to hour, I will fix it